raisinbread/README.md

```diff
@@ -44,31 +44,30 @@ in the database. Ensure that a backup is available if this data is important:
 
 ```bash
 cat raisinbread/instruments/instrument_sql/9999-99-99-drop_instrument_tables.sql \
-    SQL/9999-99-99-drop_tables.sql | mysql
+    SQL/9999-99-99-drop_tables.sql | mysql -u admin_username -p database_name
 ```
+
 ***Note:** This command can also be used at any step to empty and delete all RaisinBread tables.*
 
 ***Important:** Ensure that the above commands were completed properly and that all the tables were dropped before continuing the installation process.*
 
+If you could not drop all the tables with the commands above, you have the option to drop the tables by  manually sourcing these .sql files directly 
+from the mysql command line as follows:
+```bash
+mysql> source /var/www/loris/raisinbread/instruments/instrument_sql/9999-99-99-drop_instrument_tables.sql
+mysql> source /var/www/loris/SQL/9999-99-99-drop_tables.sql
+```
+
 In order to be able to use the RaisinBread dataset, the LORIS SQL schema needs to be
 sourced, followed by the different instrument schemas and finally the actual RB data.
 The commands below assume that the current working directory is the main LORIS root
 directory. If the tables were not deleted or created properly, the schemas can be sourced 
-directly on the mysql command line.
+directly on the mysql by using the `source` command.
```

Comment:
wondering - why add `source` if it's not mentioned in the example below?

Reply:
this is an unnecessary change